controllers/controller.js

```diff
@@ -21,4 +21,17 @@ function retrieveArticleById(req, res, next) {
     })
 }
 
-module.exports = { retreiveTopics, retrieveArticleById };
\ No newline at end of file
+function retrieveArticles(req, res, next) {
+    getArticlesSorted()
+    .then((result) => {
+        const articlesToSend = result.map((article) => {
+            delete article.body;
```

Comment:
you shouldn't be deleting anything here, you should just use the model to only select what you want